projects/golang/go/Makefile

```diff
@@ -55,9 +65,10 @@ endif
 golang-build: check-env setup-rpm-tree fetch-golang-source-archive copy-sources-to-rpmbuild-tree copy-patches-to-rpmbuild-tree build-golang-rpm generate-golang-archive test
 
 .PHONY: build
-build: golang-build sync-artifacts-to-s3-dry-run local-images
-build:
-	echo "test"
+build: golang-build sync-artifacts-to-s3-dry-run
+ifeq ($(BUILD_PREPROD_IMAGES),true)
```

Comment:
i would prefer not using ifeq, instead use a inline if in the build target above or setup a var at the top defining these targets, similar to https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L401

Reply:
yeah, happy to do it that way. other than consistency, any reason you don't like the make `ifeq`?

Reply:
done